task5/servlets/NameServlet.java

```diff
@@ -0,0 +1,30 @@
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet("/get")
+public class NameServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws IOException {
+        String name = request.getParameter("name");
+        PrintWriter pw = response.getWriter();
+
+        pw.println("<html>");
```

Comment:
Technically, `<html>` redundant here because does not make up the "complete" markup anyway) For a complete markup, you would need a doctype tag, then, the <head> and <body>, etc.

Reply:
Thanks for your corrections!
I hope that everything you said I did right.
Can I merge or are there more inaccuracies?